.github/workflows/deploy-main-on-staging.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow automatically deploys main on staging
+name: "Deploy `main` automatically on staging"
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Maybe we can do it using a tag instead automatically? Just to have better control on it.

Reply:
I think automatically would make things easier: we'll always know that the latest version runs there.